test/test_framework_test.go

```diff
@@ -538,14 +538,14 @@ func TestImportBuiltinContracts(t *testing.T) {
 	`
 
 	scriptCode := `
-	    import FungibleToken from "FungibleToken"
-	    import FlowToken from "FlowToken"
-	    import FUSD from "FUSD"
-	    import NonFungibleToken from "NonFungibleToken"
-	    import MetadataViews from "MetadataViews"
-	    import ExampleNFT from "ExampleNFT"
-	    import NFTStorefrontV2 from "NFTStorefrontV2"
-	    import NFTStorefront from "NFTStorefront"
+	    import "FungibleToken"
```

Comment:
Instead of changing the tests, could you please test both cases that are now supported, when `from` occurs and when it does not?

Reply:
@turbolent This behavior is actually tested here https://github.com/onflow/cadence-tools/blob/8b12d4d3109a71725fe8ea138e25e939ba9d6426/test/test_framework_test.go#L2799. That's why I changed the test case to the specific issue that I found. Do you think they suffice?

Reply:
Ah, great!